diff --git a/.env.development b/.env.development
index 9cd4cba..dfc0077 100644
--- a/.env.development
+++ b/.env.development
@@ -6,9 +6,9 @@ VITE_SUBMIT_ENDPOINT=https://submit-api-test.neosku.workers.dev/api/submit
 VITE_PACKAGE_STATE_ENDPOINT=https://package-state.neosku.workers.dev/v1/event
 
 # カタログデータ(テスト用)
-# VITE_REMOTE=./test/sample/index.json
+VITE_REMOTE=./test/sample/index.json
 # カタログデータ(本番用)
-VITE_REMOTE=https://raw.githubusercontent.com/Neosku/aviutl2-catalog-data/main/index.json
+# VITE_REMOTE=https://raw.githubusercontent.com/Neosku/aviutl2-catalog-data/main/index.json
 
 # セットアップ設定
 VITE_SETUP_REMOTE=./test/setup.json
